index.js

```diff
@@ -17,14 +24,11 @@ var _ = require('underscore');
 function Heartbeat(baseUrl, pulse = 60) {
 
   // validate
-  if (_.isUndefined(baseUrl)) return console.error('Url of heartbeat service is missing');
-  if (!_.isNumber(pulse) || _.isNaN(pulse) || pulse < 0) return console.error('Pulse must be positive integer');
-
   // init
   this._lastPulse = {};
 
   // set
-  this._pulse = pulse;
+  this._pulse = 900;
```

Comment:
why 900?

Reply:
15 mins is often enough for heartbeats to pulse. Id actually reduce this even more. Don't want to spam zelda with unneeded pulses. Especially since one pulse a day should be more than enough.